<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>7500</width>
    <height>5600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inteligencia Artificial - Estrategias de Búsqueda</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="gv_Mapa">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="form_Panel">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QGroupBox" name="group_Escenario">
        <property name="minimumSize">
         <size>
          <width>229</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Escenario</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>279</width>
           <height>138</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Escenario">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Tamano">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_FilCol">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_Fil">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <item>
                 <widget class="QLabel" name="lbl_Tamano">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Tamaño:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_Filas">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="value">
                   <number>25</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lbl_Fil">
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Fil</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_Col">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <item>
                 <widget class="QSpinBox" name="spin_Columnas">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="value">
                   <number>25</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lbl_Col">
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Col</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Escala">
            <item>
             <widget class="QLabel" name="lbl_Escala">
              <property name="text">
               <string>Escala:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Aumentar">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Aumentar</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_SetEscala">
              <item>
               <widget class="QPushButton" name="btn_Reducir">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Reducir</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btn_GenerarMapa">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Generar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="group_Obstaculos">
        <property name="minimumSize">
         <size>
          <width>229</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>229</width>
          <height>280</height>
         </size>
        </property>
        <property name="title">
         <string>Obstáculos</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>172</width>
           <height>97</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btn_Inicio">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Inicio</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Final">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Final</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Nada">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Nada</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_Edificio">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Edificio</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Cono">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Cono</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Arbol">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Árbol</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>190</width>
           <height>120</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="btn_Aleatorio">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Aleatorio</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSlider" name="slider_Aleatorio">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btn_GenerarAleatorio">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Generar</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QProgressBar" name="progress_Aleatorio">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="btn_Manual">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Manual</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="group_Resolver">
        <property name="minimumSize">
         <size>
          <width>229</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Resolver</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>121</width>
           <height>62</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_Solucion">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Solucion</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Paso">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Paso a paso</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Opciones">
    <property name="title">
     <string>Opciones</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Ayuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="action_Ayuda"/>
    <addaction name="action_Acerca_de"/>
   </widget>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="action_Nuevo"/>
    <addaction name="action_Abrir"/>
    <addaction name="separator"/>
    <addaction name="action_Guardar"/>
    <addaction name="action_Guardar_como"/>
    <addaction name="separator"/>
    <addaction name="action_Salir"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menu_Opciones"/>
   <addaction name="menu_Ayuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Ayuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="action_Acerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="action_Guardar_como">
   <property name="text">
    <string>Guardar como</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="action_Guardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="action_Abrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="action_Nuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
